integration-tests/cassandra-client/pom.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-universe-integration-tests</artifactId>
+    <version>999-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>quarkus-universe-integration-tests-cassandra-client</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-universe-integration-tests-rpkgtests</artifactId>
+      <version>${project.version}</version>
+      <type>pom</type>
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>com.datastax.oss.quarkus</groupId>
+      <artifactId>cassandra-quarkus-integration-tests</artifactId>
+      <version>${quarkus-cassandra-client.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.datastax.oss.quarkus</groupId>
+      <artifactId>cassandra-quarkus-integration-tests-rpkgtests</artifactId>
+      <version>${quarkus-cassandra-client.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- Need to force version 4.2, 4.3 has binary incompatibility issues -->
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <version>4.2.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
What kind of incompatibility issues are there?
This is alarming since here you are walking around the platform BOM to get the TS to pass. But when a user creates a project using the platform BOM, the platform version of the dependency will be used instead.

Reply:
I think it's probably just that their ITs are still written with Rest Assured 4.2. I wouldn't expect it to be used at runtime. Am I right?

Reply:
Right, it's most probably the case. Still will be good clarify. This is the default for Quarkus apps.

Reply:
You are both correct, it's certainly us using something that was removed in Rest Assured 4.3. More specifically the error is:

```
java.lang.AbstractMethodError: io.restassured.internal.ResponseParserRegistrar.getProperty(Ljava/lang/String;)Ljava/lang/Object;
	at org.codehaus.groovy.runtime.callsite.PogoGetPropertySite.getProperty(PogoGetPropertySite.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGroovyObjectGetProperty(AbstractCallSite.java:309)
	at io.restassured.internal.ResponseParserRegistrar.<init>(ResponseParserRegistrar.groovy:44)
	at io.restassured.RestAssured.createTestSpecification(RestAssured.java:1448)
	at io.restassured.RestAssured.given(RestAssured.java:653)
...
```

However it wasn't easy to find the incriminated method call because there is some groovy involved, and as you pointed out, this is a test dependency only, and we don't impose it on users of our extension, so eventually we gave up on trying to fix this for now.

Maybe @absurdfarce has more details.

Reply:
Ok, that's good enough. Thanks.

Reply:
FTR there's some additional detail on the issues around this test in https://github.com/datastax/cassandra-quarkus/issues/28